src/db.ts

```diff
@@ -461,46 +479,39 @@ export const saveCardReference = async (cardRef) => {
 
   const cardReferences = await getCardReferences();
   cardReferences.push(cardRef);
-  await redisClient.setAsync(
+  await redisClient.set(
     `${process.env.MOCKSOLARIS_REDIS_PREFIX}:cardReferences`,
     JSON.stringify(cardReferences)
   );
 
   return true;
 };
 
-export const getCardData = async (cardId) => {
-  const persons = await getAllPersons();
-
-  const cardData = _(persons)
-    .map((person) => _.get(person, "account.cards", []))
-    .flatten()
-    .value()
-    .find((cd) => cd.card.id === cardId);
-
-  return cardData;
+export const getCardData = async (cardId: string): Promise<Card> => {
+  const personWhoOwnsTheCard = await findPerson(
+    (p) => !!(p?.account?.cards || []).find((cd) => cd?.card?.id === cardId)
```

Comment:
:thought_balloon: you can use `some` instead of `find`.

Reply:
Related commit: https://github.com/kontist/mock-solaris/pull/114/commits/152f423bbbad68c6d888adbf64adfb3c24ed0e74